components/FormInput.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
```

Comment:
here should be at first the template as we said

Reply:
check also all the components please but the first tag should be the `template`, after the `script` and after the `style`